CMakeLists.txt

```diff
@@ -212,6 +215,11 @@ set(ENABLE_OSCAP_UTIL_VM TRUE CACHE BOOL "enables oscap util VM utility")
 # ENABLES OSCAP UTIL CHROOT
 set(ENABLE_OSCAP_UTIL_CHROOT TRUE CACHE BOOL "enables oscap util chroot")
 
+# ENABLES TESTS
+# Tests will be turned on only on Linux, because the test suite uses bash
+# and other Linux-specific tools.
+set(ENABLE_TESTS ${IS_LINUX} CACHE BOOL "enables tests")
```

Comment:
but aren't the same tools available on BSD, MacOSX, Solaris, etc?

Reply:
No, because with Cross compilation it finds bash.